engine/apps/webhooks/tasks/trigger_webhook.py

```diff
@@ -172,7 +189,12 @@ def make_request(webhook, alert_group, data):
 
 
 @shared_dedicated_queue_retry_task(
-    autoretry_for=(Exception,), retry_backoff=True, max_retries=1 if settings.DEBUG else 3
+    autoretry_for=(Exception,),
+    # This allows to exclude some exceptions that match autoretry_for but for which you don’t want a retry.
+    # https://docs.celeryq.dev/en/stable/userguide/tasks.html#Task.dont_autoretry_for
+    dont_autoretry_for=(requests.exceptions.Timeout,),
```

Comment:
Catching this error will catch both `requests.exceptions.ConnectTimeout` and `requests.exceptions.ReadTimeout` errors ([source code](https://requests.readthedocs.io/en/latest/_modules/requests/exceptions/)):
- `requests.exceptions.ConnectTimeout`: "The request timed out while trying to connect to the remote server"
- `requests.exceptions.ReadTimeout`: "The server did not send any data in the allotted amount of time"

both of these seem out of our control